new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItScaleMiiDomainNginx.java

```diff
@@ -196,19 +214,58 @@ public void testScaleClustersWithRestApi() {
     }
   }
 
+  @Test
+  @DisplayName("Verify scale each cluster of the domain by using WLDF policy")
+  public void testScaleClustersWithWLDF() {
+    for (int i = 1; i <= NUMBER_OF_CLUSTERS; i++) {
+
+      String clusterName = CLUSTER_NAME_PREFIX + i;
+      curlCmd = generateCurlCmd(clusterName, SAMPLE_APP_CONTEXT_ROOT);
+
+      // scale up the cluster by 1 server
+      logger.info("Scaling cluster {0} of domain {1} in namespace {2} from {3} servers to {4} servers.",
+          clusterName, domainUid, domainNamespace, replicaCount, replicaCount + 1);
+      List<String> managedServersBeforeScale = listManagedServersBeforeScale(clusterName, replicaCount);
+      scaleClusterAndVerifyWithWLDF(clusterName, replicaCount, replicaCount + 1,
+          managedServersBeforeScale, "scaleUp");
+
+      // scale down the cluster by 1 server
+      logger.info("Scaling cluster {0} of domain {1} in namespace {2} from {3} servers to {4} servers.",
+          clusterName, domainUid, domainNamespace, replicaCount + 1, replicaCount);
+      managedServersBeforeScale = listManagedServersBeforeScale(clusterName, replicaCount + 1);
+      scaleClusterAndVerifyWithWLDF(clusterName, replicaCount + 1, replicaCount,
+          managedServersBeforeScale, "scaleDown");
+    }
+  }
+
   /**
```

Comment:
Remove TODO comment

Reply:
The TODO comment is added by Mark's request.

Reply:
Refering to  "TODO: remove this after Sankar's PR is merged "   Sankar PR is merged long time back.

Reply:
Sankar's PR did not remove NGINX.